samples/csharp/MicrosoftGraphSDK/Module.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Graph.Beta;
+using Microsoft.Graph.Beta.Models;
+
+namespace MicrosoftGraphSDK
+{
+    /// <summary>
+    /// Graph SDK endpoints for modules
+    /// </summary>
+    public class Module
+    {
+        /// <summary>
+        /// Creates a Module with given Display Name and Description
+        /// </summary>
+        /// <param name="client">Microsoft Graph service client</param>
+        /// <param name="classId">User class id</param>
+        /// <param name="displayName">Display Name</param>
+        /// <param name="description">Description</param>
+        /// <returns>EducationModule</returns>
+        public static async Task<EducationModule> CreateAsync(
+            GraphServiceClient client,
+            string classId,
+            string displayName,
+            string description)
+        {
+            try
+            {
+                var requestBody = new EducationModule
+                {
+                    DisplayName = displayName,
+                    Description = description,
+                };
+
+                return await client.Education
+                    .Classes[classId]
+                    .Modules.PostAsync(requestBody);
+            }
+            catch (Exception ex) {
+                throw new GraphException($"CreateAsync call: {ex.Message}", ex, classId, displayName, description);
+            }
+        }
+
+        /// <summary>
+        /// Post the Resources of Module
```

Comment:
nit: Post the resource under a given module.

Reply:
updated the change of comment